catalyst_voices/packages/catalyst_voices_models/pubspec.yaml

```diff
@@ -12,7 +12,6 @@ dependencies:
   convert: ^3.1.1
   ed25519_hd_key: ^2.3.0
   equatable: ^2.0.5
-  flutter_quill: ^10.5.13
```

Comment:
few lines below i assed `uses-material-design: true`. Could you try removing it now that we dont have quill here?

Reply:
and any other packages